UnityProject/Assets/Scripts/Closets/ClosetControl.cs

```diff
@@ -131,6 +131,12 @@ private void SetIsLocked(bool value)
 		}
 	}
 
+	public IEnumerator ChangeSpriteDelayed()
+	{
+		yield return WaitFor.EndOfFrame;
```

Comment:
why it needs a delay? In byond the spawn(0) caused a lot of problems which is basically a small wait and Id prefer to avoid them

Reply:
otherwise player disappearance/appearance is always one frame late, didn’t dig too deep to learn why in order to keep sanity